ngraph_creator/operations/include/Unidirectional_Sequence_RNN.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <OperationsBase.hpp>
+
+namespace android {
+namespace hardware {
+namespace neuralnetworks {
+namespace nnhal {
+
+class Unidirectional_Sequence_RNN : public OperationsBase {
```

Comment:
I think in the code we are following 2 naming conventions.. We need to align with the single naming convention